### DISCUSSION TITLE
recognise faces on the web version?

### DISCUSSION THREAD

**Discussion by vthom56** [ente-io/ente#4005] [Enhancements]
Hello everyone

I don't know if this function exists on the web version, I can't find it anyway...
Is it possible, like mobile app, to recognise faces/persons on the web version of ente?
And if so, how do you activate this function?

Thanks a lot!

**Comment by NeverRIEght** [comment]
Hello!

Image recognition is a feature, which requires a lot of resources to run. I think that the reason this feature is disabled in Web is resources usage. I am assure that the browsers has algorithms to prevent tabs to use a lot of processing power, which is not the case while using desktop application. Ente Photos desktop is using Electron, which simply speaking is a "browser" under the hood. But, as you can guess, it does not limit anything.

I am not from the Ente team and this is only guesses after all :)

**Comment by straleks** [comment]
This would be a very nice feature to have.
